src/ui/public/filter_bar/filter_pill/filter_pill.html

```diff
@@ -69,12 +69,18 @@
     <button
       class="action filter-edit"
       ng-click="pill.onEditFilter(pill.filter)"
+      ng-disabled="pill.isControlledByPanel()"
       ng-focus="pill.activateActions()"
       ng-blur="pill.deactivateActions()"
       aria-label="Edit filter"
       data-test-subj="editFilter"
     >
-      <i class="fa fa-fw fa-edit"></i>
+      <i
+        ng-show="pill.isControlledByPanel()"
+        tooltip="Edit disabled because filter is controlled by panel"
+        class="fa fa-fw fa-edit fa-disabled"
```

Comment:
One minor nitpick: is there a way we can describe this without using the word panel? I can imagine us using the `controlledBy` meta property for more than just input controls and "panel" only makes sense inside dashboard. Maybe `isControlledByKibana` and "Editing is disabled because this filter is controlled by Kibana"? Open other ideas as well.

Reply:
@cjcenizal You have been on a role with good UI wording. Any ideas on how to phrase the is controlled by message in a more generic fashion?

Reply:
To satisfy Matt's use-case then it really sounds like the controlling element should be responsible for providing this message to the pill, e.g. in the case of the input controls `\`Edit filter using the "${panel.title}" controls\``. So if the input control panel is labelled "Population density", then the tooltip would read `Edit filter using the "Population density" controls.` That way we can customize the format of this message depending on what kind of control is controlling that pill.

Reply:
And minor nit, we should be using `<span>` elements for icons instead of `<i>` (which is actually a deprecated part of the HTML4 spec).

Reply:
I was thinking about this some more, and the input control (and other filter-controlling UI elements) should probably just surface a property called something like `humanizedTitle`, which in my above example would be:

```js
`${panel.title}" controls`
```

Then the filter itself should be responsible for formatting the message using this value, so that it surfaces the message in a consistent format:

```js
`Edit filter using the ${meta.humanizedTitle}.`
```

Reply:
@bargs What do you think about the adding `humanizedTitle` to meta? Does all of the meta stuff get added to the URL?

Reply:
Yeah, everything in meta goes in the URL and gets persisted in the SavedObject. It feels a little weird but I can't think of a really solid reason not to do it. 

I am a little worried `humanizedTitle` could be mistaken for the filter's title rather than the controller's title though. Perhaps we should make the value of `controlledBy` an object so that it could look like this:

```
{
  meta: {
    controlledBy: {
      id: 1234,
      title: Foo Panel
    }  
  }
}
```

Reply:
What about cases where the visualization is still being created and is not saved and thus unnamed?

Reply:
I guess it would just use the default title, "New Visualization (unsaved)". Not pretty, but it's an edge case anyway right? One downside to putting the title in the meta though is that it can get stale. If a user created a filter during viz construction like you said and pinned it, it would be stuck with that name. But maybe that's an edge case we don't care about. I'm not sure if there's a perfect solution with the existing filter bar, I think long term we're probably going to need to figure out a better way of displaying these "managed" filters.

Reply:
Even worse, if the filter is pinned and then switches to another dashboard - the filter is there but now the controlling panel is not. So the only thing the user can do is delete the filter.

Reply:
Would it be possible to "release control" at that point?

Reply:
Or maybe disable pinning if it's controlled?

Reply:
It is very easy to also disable pinning when a filter is controlled if thats desired